patches/chromium/export_fetchapi_mojo_traits_to_fix_component_build.patch

```diff
@@ -0,0 +1,36 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jeremy Apthorp <jeremya@chromium.org>
+Date: Fri, 20 Sep 2019 16:44:18 -0400
+Subject: export FetchAPI mojo traits to fix component build
+
+Without these, we get link errors in the component build when using the
+blink::CloneableMessage mojo traits.
```

Comment:
Is there a CL for this ?

Reply:
Not yet, but I talked to pfeldman@ and he said he'd approve it if i uploaded it :P will do soon.